anagram.cpp

```diff
@@ -1,13 +1,64 @@
 #include "anagram.h"
+using namespace std;
+
+void removeSpace(std::string& w) {
```

Comment:
Looks like removeSpace is not used. Remove unused functions. Also remove commented calls to this function.